bin/run-dev.sh

```diff
@@ -2,4 +2,14 @@
 
 urlwait
 python manage.py migrate --noinput
-python manage.py runserver 0.0.0.0:8000
+
+granian --interface wsgi \
+    --host 0.0.0.0 \
+    --port "${PORT:-8000}" \
+    --no-ws \
+    --workers "1" \
+    --threads "1" \
+    --log-level "${GRANIAN_LOG_LEVEL:-debug}" \
+    --access-log \
+    --reload \
+    basket.wsgi:application
```

Comment:
I could be convinced not to go this route, but I do like it to flex granian while developing. I tested the reload and it works well -- even for template changes.